### DISCUSSION TITLE
Where could I disable startup sounds on G-Helper

### DISCUSSION THREAD

**Discussion by cpbrr** [seerge/g-helper#664] [Q&A]
Recently I'd installed a new bios so that must have reseted the option for startup sounds. Where could I disable it? If it's not in G-Helper can you maybe implement it in the future?

**Comment by seerge** [comment]
Hello, you can disable it in bios (f2 on boot)
https://github.com/seerge/g-helper/issues/261

**Reply by seerge** [reply]
Added in https://github.com/seerge/g-helper/releases/tag/v0.120
